backend/dal/Services/Concrete/PropertyService.cs

```diff
@@ -0,0 +1,146 @@
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using Pims.Dal.Entities.Views;
+using Pims.Dal.Helpers.Extensions;
+using Pims.Dal.Security;
+using System;
+using System.Linq;
+using System.Security.Claims;
+using Pims.Core.Extensions;
+using Pims.Dal.Entities.Models;
+
+namespace Pims.Dal.Services
+{
+    /// <summary>
+    /// PropertyService class, provides a service layer to interact with properties within the datasource.
+    /// </summary>
+    public class PropertyService : BaseService, IPropertyService
+    {
+        #region Constructors
+        /// <summary>
+        /// Creates a new instance of a PropertyService, and initializes it with the specified arguments.
+        /// </summary>
+        /// <param name="dbContext"></param>
+        /// <param name="user"></param>
+        /// <param name="logger"></param>
+        public PropertyService(PimsContext dbContext, ClaimsPrincipal user, ILogger<PropertyService> logger) : base(dbContext, user, logger) { }
+        #endregion
+
+        #region Methods
+        /// <summary>
+        /// Get an array of parcel properties within the specified filters.
+        /// Will not return sensitive properties unless the user has the `sensitive-view` claim and belongs to the owning agency.
+        /// </summary>
+        /// <param name="filter"></param>
+        /// <returns></returns>
+        public Paged<Property> Get(ParcelFilter filter)
+        {
+            this.User.ThrowIfNotAuthorized(Permissions.PropertyView);
+
+            var query = this.Context.GenerateQuery(this.User, filter)
+                    .Include(p => p.Status)
+                    .Include(p => p.Classification)
+                    .Include(p => p.Agency)
+                    .Include(p => p.Address)
+                    .ThenInclude(a => a.City)
+                    .Include(p => p.Address)
+                    .ThenInclude(a => a.Province);
+
+            var total = query.Count();
+            var items = query.Select(p => new Property(p)
+            {
+                Status = p.Status.Name,
+                Classification = p.Classification.Name,
+                AddressId = p.AddressId,
+                Address = $"{p.Address.Address1} {p.Address.Address2}",
```

Comment:
This is quite minor but if `Address.Address2` is empty (which is quite common as most addresses only have one line) there would be an extra space at the end...

Maybe checking for not-empty before appending `Address2`?

Reply:
I can't do the trim or the query will not run in the database regrettably.

Reply:
okay no big deal anyways